misc/services/mobile-webui/mobile-webui-frontend/src/api/index.js

```diff
@@ -1,5 +1,14 @@
+import axios from 'axios';
+
 import {
     getLaunchers
 } from './launchers';
 
-export { getLaunchers };
\ No newline at end of file
+function loginRequest(username, password) {
+  return axios.post(`/app/api/v2/auth`, {
```

Comment:
let's have the `/app/api/v2` part of the BASE URL config.

Reply:
@petrican  @siemiatj  wdyt?

Reply:
Yeah that is mandatory to use the ${window.config.API_URL} , otherwise since this is hosted on https://devxxx.metasfresh.com/mobile it will load from the root and you don't have the guarantee that APIs are going to be found in that path. Also while developing on localhost you need a way to setup the remote API so I believe this would be the best approach.